### PR TITLE
Split search indices

### PR DESCRIPTION
**Pull request by GLindebyV**

### CODE REVIEW

File: combind.go [hunk 1]
```diff
@@ -18,8 +18,8 @@ type Combind struct {
 }
 
 type CombinerBuilder interface {
-	Update(ctx context.Context, comp ...*Component) ([]*SearchBox, error)
-	Save(ctx context.Context) ([]*SearchBox, error)
```
Comment (marbergq) [inline combind.go:22]: What is withProps? Why is that needed here?

File: combind.go [hunk 2]
```diff
@@ -18,8 +18,8 @@ type Combind struct {
...
+	Update(ctx context.Context, withProps map[string]interface{}, index string, comps ...*BackendComponent) ([]*SearchBox, error)
```
[the remark this answers is not in the document]
Comment (GLindebyV) [reply combind.go:21]: With props specifies which components should be built out. The index specifies which is the target index to store the built components
